app-misc/qlcplus/qlcplus-4.12.2.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils udev virtualx xdg
+
+DESCRIPTION="A software to control DMX or analog lighting systems"
+HOMEPAGE="https://www.qlcplus.org/"
+SRC_URI="https://github.com/mcallegari/${PN}/archive/QLC+_${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Why are you renaming it? The original name looks fine.

Reply:
That was copied from the ebuild currently in the portage tree, I just didn't bother to change. Of course we can leave away the renaming if it doesn't break anything. Will investigate and change

Reply:
Changed ebuilds to not rename the distfile and confirmed that it works properly